main.tf

```diff
@@ -137,7 +164,8 @@ resource "azurerm_windows_function_app" "windows_function" {
   client_certificate_enabled  = var.client_certificate_enabled
   client_certificate_mode     = var.client_certificate_mode
   builtin_logging_enabled     = var.builtin_logging_enabled
-  virtual_network_subnet_id   = length(var.subnet_name) == 0 ? azurerm_subnet.main[0].id : var.subnet_id
+  virtual_network_subnet_id   = length(var.subnet_name) == 0 ? azurerm_subnet.main[0].id : data.azurerm_subnet.main.0.id
+  public_network_access_enabled = false
```

Comment:
what will happen if it is a standard sku ? shouldn't this be true in that case ?